autoload/ale/completion.vim

```diff
@@ -902,16 +902,17 @@ endfunction
 
 " This function implements the :ALEImport command.
 function! ale#completion#Import() abort
-    let l:word = expand('<cword>')
+    let l:wordpos = ale#util#GetCWordBoundary()
+    let l:word = wordpos[0]
 
     if empty(l:word)
         call s:message('Nothing to complete at cursor!')
 
         return
     endif
 
-    let [l:line, l:column] = getpos('.')[1:2]
-    let l:column = searchpos('\V' . escape(l:word, '/\'), 'bn', l:line)[1]
+    let l:line = getpos('.')[1]
+    let l:column = l:wordpos[2]
```

Comment:
If we need the end position of the word, can't we just use `len(expand('<cword>'))` to work out the end position based on the start position we have already?

Reply:
Well the vim function `matchstrpos` (used in `ale#util#GetCWordBoundary()`) returns the start and end position of the match, so we already have the end position.

The line (in GetCWordBoundary),
```
let lastpos = matchstrpos(getline('.'), escape(l:word, '/\'), laststart)
```
has `lastpos` set to [word, startpos, endpos].

We could, and then have just one return value, and change the function name `GetCWordBoundary` to `GetCWordStartColumn`.

What do you think?

Reply:
Can't we just use `len(expand('<cword>'))` to know how long the string is and use that to get the end position from the start position we already know?

Reply:
I think I see what you are asking. Yes, if you add 'c' to the flags for `searchpos()`, then that should get the beginning position, and we could use `len(expand('<cword>')` to get the end position. That would work.

All depends if you want to use `matchstrpos()` or `searchpos()`. Do you think there is a significant performance difference between the two?

Reply:
I don't think it will matter.